api/numaresourcesoperator/v1alpha1/numaresourcesoperator_types.go

```diff
@@ -43,21 +44,35 @@ type NUMAResourcesOperatorSpec struct {
 	PodExcludes []NamespacedName `json:"podExcludes,omitempty"`
 }
 
-// +kubebuilder:validation:Enum=Enabled;Disabled
+// +kubebuilder:validation:Enum=Disabled;Enabled;EnabledExclusiveResources
 type PodsFingerprintingMode string
 
-var (
+const (
 	// PodsFingerprintingDisabled disables the pod fingerprinting reporting.
 	PodsFingerprintingDisabled PodsFingerprintingMode = "Disabled"
 
 	// PodsFingerprintingEnabled enables the pod fingerprint considering all the pods running on nodes. It is the default.
 	PodsFingerprintingEnabled PodsFingerprintingMode = "Enabled"
+
+	// PodsFingerprintingEnabledExclusiveResources enables the pod fingerprint considering only pods which have exclusive resources assigned.
+	PodsFingerprintingEnabledExclusiveResources PodsFingerprintingMode = "EnabledExclusiveResources"
+)
+
+// +kubebuilder:validation:Enum=Disabled;Enabled
+type InfoRefreshPauseMode string
+
+const (
+	// InfoRefreshPauseDisabled enables RTE and NRT sync
+	InfoRefreshPauseDisabled InfoRefreshPauseMode = "Disabled"
+
+	// InfoRefreshPauseEnabled pauses RTE and disables the NRT sync
+	InfoRefreshPauseEnabled InfoRefreshPauseMode = "Enabled"
 )
```

Comment:
I understand that PodsFingerprintingMode was present in v1alpha1 before and starting from 4.16 it had new option, but why to include new fields like InfoRefreshPause in v1alpha1 that were never present (meant to be supported) in v1alpha1?

Reply:
Good point. I was not completely sure about this, so let's go incrementally and let's minimize this change.